.circleci/config.yml

```diff
@@ -0,0 +1,23 @@
+# Python CircleCI 2.0 configuration file
```

Comment:
I know this shows up on circleCI app - I'm curious why it isn't rendering on GitHub